Web/starter_project_iconics/pages/blog/index.tsx

```diff
@@ -1,10 +1,15 @@
 import BlogTile from '@/components/blog/BlogTile'
-import { Blog } from '@/types/blog/blog'
+import { useGetBlogsQuery } from '@/store/features/api/blog-api'
+import Link from 'next/link'
 import { FaPlus } from 'react-icons/fa'
-import data from '../../data/blog/blog.json'
 
-const blogs = () => {
-  const blogsData: Blog[] = data
+const Blogs = () => {
+  const { data: blogsData, isLoading } = useGetBlogsQuery()
+  console.log(blogsData)
```

Comment:
it's important to ensure that any debugging statements, including console.log, are removed before committing the code.